File: benchmark/results/python-invocation-script/run-003/a-mcp/output/shipping/services/shipment_repository.py
from shipping.aggregates.shipment import Shipment

"""ShipmentRepository service."""
class ShipmentRepository:
    def create(self, input: Shipment) -> Shipment:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> Shipment | None:
        raise NotImplementedError('not implemented')
    def list(self, limit: float) -> list[Shipment]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

File: benchmark/results/java-multilang/run-004/b-baseline/output/com/metaengine/demo/shipping/services/ShipmentRepository.java
package com.metaengine.demo.shipping.services;

import java.util.List;

import com.metaengine.demo.shipping.aggregates.Shipment;

/** ShipmentRepository persistence service. */
public class ShipmentRepository {

    /** Persist a Shipment from a partial input. */
    public Shipment create(Shipment input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a Shipment by id (may return null). */
    public Shipment findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List shipments up to limit. */
    public List<Shipment> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a Shipment by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
